valhalla/sif/autocost.h

```diff
@@ -4,24 +4,47 @@
 #include <cstdint>
 
 #include <boost/property_tree/ptree.hpp>
+#include <valhalla/baldr/rapidjson_utils.h>
+#include <valhalla/proto/directions_options.pb.h>
 #include <valhalla/sif/dynamiccost.h>
 
 namespace valhalla {
 namespace sif {
 
+/**
+ * Parses the auto cost options from json and stores values in pbf.
+ */
+void ParseAutoCostOptions(const rapidjson::Document& doc,
+                          const std::string& costing_options_key,
+                          odin::CostingOptions* pbf_costing_options);
+
```

Comment:
Comment the parameters? In particular what does the costing_options_key represent?

Reply:
I can add comments. The `costing_options_key` is the json key from where you will parse the information in the specified doc

Reply:
@dnesbitt61 I added param comments